test_fms/fms2_io/test_io_simple.F90

```diff
@@ -0,0 +1,90 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+! This is a very simple test of FMS IO.
+
+! Ed Hartnett, 6/10/20
+
+program test_io_simple
+  use, intrinsic :: iso_fortran_env, only : real32, real64, int32, int64, error_unit, output_unit
+  use mpi
+  use fms2_io_mod
+  use netcdf_io_mod
+  use mpp_domains_mod
+  use mpp_mod
+  use setup
+  use netcdf
+  
+  type(Params) :: test_params
+  type(domain2d) :: domain
+  integer :: err
+  type(domain2d), pointer :: io_domain
+  type(FmsNetcdfDomainFile_t) :: fileobj
+  integer, parameter :: ntiles = 6
+  integer :: ncchksz = 64*1024 
+  character (len = 10) :: netcdf_default_format = "64bit"
+  integer :: header_buffer_val = 16384
+  integer :: ncid
+
+  ! Initialize.
+  call init(test_params, ntiles)
```

Comment:
Does this come from the `setup` module?  Is it calling mpp_init?

Reply:
Yes and yes.